utils/string.go

```diff
@@ -63,6 +63,36 @@ func ShortenUUID8(uuid string) (string, error) {
 	return hex.EncodeToString(returnBytes), nil
 }
 
+func ShortenHexString(input string, outputLen int) (string, error) {
```

Comment:
This totally makes sense, love the code, can u document the function itself and write when would we need to use this